Library/Homebrew/rubocops/lines.rb

```diff
@@ -665,6 +665,21 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
         end
       end
 
+      # This cop ensures that new formulae depending on Requirements are not introduced in homebrew/core.
+      class CoreRequirements < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, _body_node)
+          if depends_on? :java
```

Comment:
```suggestion
        def audit_formula(_node, _class_node, _parent_class_node, _body_node)
          return if formula_tap != "homebrew-core"

          if depends_on? :java
```

Reply:
Whoops, dunno how that vanished from my branch...